webpack.config.js

```diff
@@ -47,6 +48,7 @@ module.exports = {
   devServer: {
     contentBase: path.join(__dirname, 'dist'),
     disableHostCheck: true,
+    historyApiFallback: true,
```

Comment:
What does this do and why is this needed?

Reply:
If I got it right, on devserver if page is not found it shall it redirected to `index.html` instead of showing 404.
I had some issues, when saving the project and page would show me 404 instead of actual project with updated url. 
Those updates fixed the issue.

Reply:
Basically, always serve "index.html", regardless of the URL. React Router decides what to show based on the URL, not the webserver.